### PR TITLE
Release 3.4.1

### PR DESCRIPTION
**Pull request by mtsvr**
- Add documentation template to serve as guide for documenting new contributions.
- Add component index with links to respective `README`s using [relative links](https://github.blog/2013-01-31-relative-links-in-markup-files/)

### CODE REVIEW

File: COMPONENTS.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Component Index
+
+In this index you will find a list containing the documented components of `phoenix-shared`. Each
+item has a link to it's correspoonding documentation.
```
Comment (raicp) [inline COMPONENTS.md]: typo: double o

File: DOCUMENTATION_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Documentation Template
+
+This is a documentation template for a component's `README.md` (remember to create it if you haven't already). Use it anytime you add a component
+or container to `phoenix-shared`.
+
+Use the the sections mentioned here as required, and try to follow the instructions for each section.
```
Comment (raicp) [inline DOCUMENTATION_TEMPLATE.md]: typo: double the

File: DOCUMENTATION_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Remember to remove the quotes when filling in your documentaiton.
```
Comment (raicp) [inline DOCUMENTATION_TEMPLATE.md]: type: documentation

File: DOCUMENTATION_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+| `limit` | The number of elements per page | `5` | `false` |
+
+## How to use
+
+> Use this section to indicate how to use the component/container that's being documented.
+> Start this section including one or more paraghraphs indicating important considerations when using
+> the component, when needed. Any warning that needs to be made, must be made here.
+
+> If the component/container has different use cases, or different items that need to be explained
+> you can use sections.
```
Comment (ignaciosy) [inline DOCUMENTATION_TEMPLATE.md]: Pondría `you can use sections` empezando el parrafo: `You can use sections if the component...`

File: README.md [hunk 5]
```diff
@@ -26,4 +26,9 @@ If you find an error, please raise an issue at GitHub's issues tab.
 
 To contribute, send a pull request pointing to `develop` branch. Be sure to leave a detailed explanation of your changes in the pull request description to make its revision easier and therefore faster.
 
-When contributing new functionality to _phoenix-shared_, try adding proper documentation to [the repo's wiki](https://github.com/Papinotas-Desarrollo/phoenix-shared/wiki)
+When contributing new functionality to _phoenix-shared_, try adding proper documentation to adding a `README.md`
```
Comment (ignaciosy) [inline README.md]: Escribiría algo como: `When contributing a new functionality to _phoenix-shared_, try adding proper documentation by including a Readme...`

Comment (mtsvr) [reply README.md]: valee, no me fije en eso que estaba de antes.

File: DOCUMENTATION_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+> and `Required?` to indicate whether the prop is required. Remember to remove the example of your prop list.
+
+| Prop        | Description        | Default value | Required? |
+| ---- | ---- | ---- | ---- |
+| `limit` | The number of elements per page | `5` | `false` |
+
+## How to use
+
+> Use this section to indicate how to use the component/container that's being documented.
+> Start this section including one or more paraghraphs indicating important considerations when using
```
Comment (ARivas16) [inline DOCUMENTATION_TEMPLATE.md]: typo `paragraphs`

**Pull request merged**